### PR TITLE
Garage: Show MB contributions on owner profile page

### PR DESCRIPTION
**Pull request by datadanne**
Closes RIG-71

### CODE REVIEW

**Review by sanderpick — DISMISSED** [review]
LGTM!

**Review by sanderpick — DISMISSED** [review]

**Comment by datadanne** [comment]
I had to rebase this PR because of a merge conflict after the merge of main -> staging. Can I get another approval? :pray:

**Review by sanderpick — APPROVED** [review]

**Pull request merged**
